frontend/src/components/Register.tsx

```diff
@@ -0,0 +1,88 @@
+import 'bootstrap/dist/css/bootstrap.min.css';
```

Comment:
Where are all these imports coming from? You never updated the package json. Did you install these packages??